x-pack/elastic-agent/pkg/remote/client.go

```diff
@@ -165,15 +168,15 @@ func (c *Client) Send(
 	params url.Values,
 	headers http.Header,
 	body io.Reader,
-) (*http.Response, error) {
+) (*http.Response, CancelFunc, error) {
```

Comment:
We should point out in the function comment that the caller is responsible for canceling it.